web-registry/src/pages/Media.tsx

```diff
@@ -51,24 +66,34 @@ const Media: React.FC = () => {
           },
         },
       });
+      if (!isEdit) navigateNext();
     } catch (e) {
       // TODO: Should we display the error banner here?
       // https://github.com/regen-network/regen-registry/issues/554
       // console.log(e);
     }
   }
 
+  const Form = (): JSX.Element => (
+    <MediaForm
+      submit={submit}
+      initialValues={initialFieldValues}
+      onNext={navigateNext}
+      onPrev={navigatePrev}
+    />
+  );
+
   return isEdit ? (
     <EditFormTemplate>
```

Comment:
we should probably refactor this at some point and extract this logic `isEdit? <EditFormTemplate>...</EditFormTemplate> : <OnboardingFormTemplate>...</OnboardingFormTemplate>` into a reusable template component... cc/ @mhagel